gitbase/values.yaml

```diff
@@ -0,0 +1,54 @@
+# Default values for gitbase.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: srcd/gitbase
+  tag: v0.17.1
+  pullPolicy: IfNotPresent
+
+service:
+  type: ClusterIP
+  port: 3306
+
+image:
```

Comment:
This section is duplicated. Remove it. Or make sure that `pullPolicy` is set to `Always` if you decide to move on with `latest`